### DISCUSSION TITLE
Question about the article Layout Builder.

### DISCUSSION THREAD

**Discussion by Banuk1616** [templaza/astroid-framework#765] [Q&A]
I have created different field groups (A,B, and C). In each of these field groups I have created fields (A1,A2,A3, B1.B2, etc.) The contents of these fields should be displayed at different positions in the article. I have created an article layout with the Article Layout Builder. So far everything is fine. How do I now manage to assign the different field groups to a position in the article layout? Currently, all the contents of the fields are displayed in a single position? Is that even possible?
I would be very grateful for any help or advice.
Best regards

**Comment by paurray** [comment]
Honestly I do not understand what you are doing.
Have you seen this. (It is in German but on YT you can Auto Translate to the language of your choice!)

Astroid Article Layout setting
https://www.youtube.com/watch?v=_OKdz8lXQ3Q

Maybe it helps?

PS I see you are also ;-) based in Germany so maybe you dont need to translate

**Comment by Banuk1616** [comment]
@paurray

First of all, thank you for your response to my question. I have seen this video because before I ask questions in the forum I try to find answers on the internet.  My native language is German and I can understand this video, but it doesn't answer my question or problem. Unfortunately my English is not very good either, it's been 45 years since I learned it at school but never had to use it afterwards. Therefore it is difficult for me to describe complex problem in English without translator. I apologize for that.

I understand how the article layout builder works, but I don't understand how I can create custom fields of different field groups in different places in the article.

See photo
![fieldgroups](https://github.com/user-attachments/assets/6b582e56-93a9-42e2-a313-d88b55d37918)


I have created different field groups and want to display them at different positions in the article. I can't find a solution and I don't know if this is possible?

Please excuse my bad English, I am overwhelmed when it comes to complicated questions.

Best regards

**Comment by paurray** [comment]
This is what I am building here:
https://finalbug.tv/en/
https://finalbug.tv/de/

**Step 01**

<img width="960" alt="ALB-Step01" src="https://github.com/user-attachments/assets/9eb543c3-7cd1-4ca1-8be7-2d5191d64227">

Use the Article Layout Builder to build a Template with Layout Content eg Text Layout is 111 and 222 and 333


**Step 02**

<img width="960" alt="ALB-Step02" src="https://github.com/user-attachments/assets/4e562221-1e7c-4917-842d-42e7eef6c776">

In the Joomla Category that my Blog Article is displayed in select Article Layout Template in my case TWO (edited) and the Article Layout set up and saved in step 01

**Step 03a & 03b**

<img width="960" alt="ALB-Step03" src="https://github.com/user-attachments/assets/98ce2443-46d0-421f-ae91-932b42c4eb2e">


Add content to articles for Home (EN &DE)
Replacing 111 with German and English content depending on whether I am on the German or English version of site.

**Step 04**

<img width="960" alt="ALB-Step04" src="https://github.com/user-attachments/assets/06871556-cdb8-4108-a961-f7a425d91134">

I added some “Lorem Ipsum” content to the standard Joomla content part of an article for the home page and it did not display.

Maybe as you are guessing it is not supposed to work this way? Maybe some one else can confirm?

Have you considered adding text blocks to your Article Layout Template? Basically using it as you would use the Layout builder, building all parts of the article including article text and widgets?

No idea if this makes any sense!

**Reply by sonvnn** [reply]
@paurray You have to add Text Addon to Article Layout display Text of Article.
Go back to Article Layout Builder you can find it below
![Screen Shot 2024-08-29 at 13 55 12](https://github.com/user-attachments/assets/22fd7e79-406f-4ba0-b789-5a5f9a4457f6)

**Reply by paurray** [reply]
@sonvnn Thank you
I will include this detail in a future Video Tutorial 💯

**Reply by sonvnn** [reply]
You are welcome :)

**Comment by Banuk1616** [comment]
@paurray
Thank you for taking the time to help me.

The problem is the user-defined fields. I have created these in several field groups and can call them up in the editor in the tabs and fill them in.

![fieldgroups2](https://github.com/user-attachments/assets/8c549a23-1b45-48ce-9410-f44915246567)

Sonny has also done this in the Two Demo, but only with one per article field group.

![fieldgroups1](https://github.com/user-attachments/assets/e08470a3-3d6d-45a9-ba92-e7be59e9af25)

I would like to display several field groups at different positions in the article. I cannot do this. All field groups are displayed at the same position in the article and I don't know if my plan is even feasible.

Best Regards

**Reply by sonvnn** [reply]
At the moment we can not display separate Field groups. But many ideas for this issue. I will consider a solution for it in the next versions. But I think it will take a lot of time because I have to dig deeper to Joomla core

**Comment by jam1974** [comment]
Maybe the following idea will work. You have to test it. Use the customfields module (https://github.com/bubi-luka/mod_customfields)  to integrate your fields into a module and then into your layout via the module widget. It's worth a try.

**Reply by Banuk1616** [reply]
@jam1975
Thank you for this idea and the link. I will test it and then give you feedback.
Best regards

**Reply by jam1974** [reply]
No modules can yet be loaded into the Article Layout Builder

**Reply by sonvnn** [reply]
Yes, At the moment you have to use Plugin - Loadmodule to load a module in the article.

**Reply by jam1974** [reply]
Unfortunately not - if I integrate a module {loadmoduleid 113} into the joomla content and integrate this content in the Article Layout Builder via the widget Fulltext, I do not see the module but {loadmoduleid 113} in the frontend.

**Reply by jam1974** [reply]
It will work with https://regularlabs.com/modulesanywhere in the Article Layout Builder with the text - widget.

**Reply by sonvnn** [reply]
Thank you for the update. I am on a vacation. I will check it when I back to office on the next Wednesday.
